UPDATE_CHECKLIST.md

```diff
@@ -30,6 +30,7 @@ This checklist is seen as a guide to update the existing deployment.
 - [ ] If new plugin builds are released
   - [ ] Double-check that the build- and release-metadata is published and updated correctly by the deploy script and contracts
   - [ ] If the plugin is used by the management DAO and the new build includes security relevant changes it must be applied immediately
+- [ ] If adding any new files to the deploy script, check the tags and func.dependencies to ensure the `00-env-check.ts` file can be run. This can be done by adding the `func.dependencies = ['Env']` to the function.
```

Comment:
We go through this checklist when deploying an update (`packages/contracts/deploye/update/to_vX.Y.Z`) to a network that we are already supporting. 
```suggestion
- [ ] Check all the tags and `func.dependencies` to ensure the `00-env-check.ts` file is executed at the beginning of the deployment.
```
Make sure to add this bullet point also to the `DEPLOYMENT_CHECKLIST.md`, which we use when we deploy to new, previously unsupported, network (`packages/contracts/deploye/new`).